Dockerfile

```
@@ -1,21 +1,22 @@
 # base Go image version.
-FROM golang:1.13.0-stretch AS build
+FROM golang:1.13.5-alpine AS build
```

let's force the alpine version as well
```suggestion
FROM golang:1.13.5-alpine3.10 AS build
```